person-directory-impl/src/main/java/org/apereo/services/persondir/support/BasePersonAttributeDao.java

```diff
@@ -193,4 +194,18 @@ public void setEnabled(final boolean enabled) {
         this.enabled = enabled;
     }
 
+    protected IPersonAttributes getSinglePerson(final Set<IPersonAttributes> people) {
+        IPersonAttributes person;
+        try {
+            person = DataAccessUtils.singleResult(people);
+        } catch (final IncorrectResultSizeDataAccessException e) {
+            logger.debug("Unexpected multiple people returned from person attribute DAO: {} : {} ", e.getClass().getName(), e.getMessage());
+            people.stream().forEach(p -> {
```

Comment:
stream() should be unnecessary. Call forEach and use a lambda.

Reply:
I think this is fixed.